build.psm1

```diff
@@ -2070,8 +2080,9 @@ function New-MSIPackage
         [string] $ProductVersion,
 
         # Product Guid needs to change for every version to support SxS install
```

Comment:
This comment should probably be changed to: Product Code needs to be different for every version